README.md

```diff
@@ -43,7 +43,7 @@ For best display effect, you should generate your diff text with `git diff -U1`
 
 The `{File[] parseDiff({string} text, {Object} [options])` named export is a wrap of [parse-diff](https://www.npmjs.com/package/parse-diff) package with some extra options:
 
-- `{boolean} stubChunk`: Whether to add a stub empty chunk at the tail of each chunk list, this can provide an extra hunk header when [customizing chunk header](#customize-chunk-header), for example, to expand code after the diff.
+- `{boolean} stubChunk`: Whether to add a stub empty hunk at the tail of each hunk list, this can provide an extra hunk header when [customizing hunk header](#customize-hunk-header), for example, to expand code after the diff.
```

Comment:
There are still have some `chunk` -> `hunk` fixes left, can you just global replace `Chunk` to `Hunk` (first letter upper cased) in `README.md`